### DISCUSSION TITLE
Allow mods to modify slider follow circle size

### DISCUSSION THREAD

**Discussion by PercyDan54** [ppy/osu#13553] [Q&A]
And I want to port a mod from another game to to lazer. To create the first part, I think we have `HitPolicy`
But I don't know how to change slider follow circle size.

**Comment by peppy** [comment]
I'm not sure why those things should be in the same mod in the first place (and can't guarantee we will accept a mod which is copying one from another game), but ignoring that...

Before offering any advice, what roadblock did you hit when trying to adjust the follow circle radius? I'm not sure it would be any different to how any other mod does anything in the game thus far. Do you have a branch with your attempt?

**Reply by PercyDan54** [reply]
Currently no. Haven't start working on it yet
